quesma/clickhouse/clickhouse.go

```diff
@@ -36,14 +36,13 @@ const (
 type (
 	// LogManager should be renamed to Connector  -> TODO !!!
```

Comment:
Maybe we should split `LogManager` into `QueryManager` and `IngestManager`?

Reply:
That might be candidate for other PR